src/menu/tray.js

```diff
@@ -60,7 +62,16 @@ export class Tray extends EventEmitter {
     this.#trayMenu.setToolTip('Rancher Desktop');
 
     // Discover k8s contexts
-    this.updateContexts();
+    try {
+      this.updateContexts();
+    } catch (err) {
+      if (err instanceof TypeError &&
+          err.message.includes("Cannot read property 'clusters' of undefined") &&
+          err.stack?.includes('loadFromFile')) {
+        electron.dialog.showErrorBox('Error reading config file',
+          `Please check your config file(s) for problems.\n\nError message: ${ err.message }`);
```

Comment:
Do we need to bubble the error up? Or will clobbering existing configs be acceptable?

Reply:
Okay, in that case we just rely on this being the only place we modify the kubeconfig to make sure we don't clobber the file we couldn't read, right?

Hopefully nobody is doing dumb things like `export KUBECONFIG=~/Documents/passwords.xlsx`…